Error: Cannot find module 'source-map-support/register'

I just installed `gen-typescript-declarations` version 1.6.0 and I got the following error when running the command:

```
gen-typescript-declarations --deleteExisting --outDir .
```

```
internal/modules/cjs/loader.js:582
    throw err;
    ^

Error: Cannot find module 'source-map-support/register'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/ppsztyc/workspace/arc/form-data-editor/node_modules/@polymer/gen-typescript-declarations/lib/cli.js:21:1)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
```

This was working perfectly with previous version.

Known workaround:

```
npm i --no-save source-map-support
```

--- a/packages/gen-typescript-declarations/package.json
+++ b/packages/gen-typescript-declarations/package.json
@@ -33,13 +33,13 @@
     "glob": "^7.1.2",
     "minimatch": "^3.0.4",
     "polymer-analyzer": "^3.0.2",
+    "source-map-support": "^0.5.9",
     "typescript": "^3.0.1",
     "vscode-uri": "^1.0.3"
   },
   "devDependencies": {
     "bower": "^1.8.2",
     "rimraf": "^2.6.2",
-    "source-map-support": "^0.5.0",
     "tsc-then": "^1.0.1"
   },
   "scripts": {

